static/js/request_detail.vue

```diff
@@ -410,27 +410,45 @@
             let fail_reason = '';
             for (let csIndex in that.observationData[observationIdx].configuration_statuses) {
               let configurationStatus = that.observationData[observationIdx].configuration_statuses[csIndex];
-              if (!_.isEmpty(configurationStatus.summary)) {
-                time_completed += configurationStatus.summary.time_completed;
-                if (fail_reason === '' && configurationStatus.summary.reason !== 'N/A') {
-                  fail_reason = configurationStatus.summary.reason;
-                }
-              }
+              // Find the configuration that goes with this configuration status
+              let configuration = that.request.configurations[0];
```

Comment:
should we maybe leave this undefined? If we don't find a configuration corresponding to the configuration status, I think I would rather raise an error in the console then have it appear fine but with the wrong configuration...

Reply:
Yes, I will do that.